CPS-????/README.md

```diff
@@ -0,0 +1,96 @@
+---
+CPS: ?
+Title: Light Client Protocols for Cardano
+Status: Open
+Category: Tools
```

Comment:
```suggestion
Category: Network
```
[suggestion is only put forward for debate: not a definite recommendation at this point]

I think it would be a good juncture to consider whether protocols for these _Use Cases_, and defined as in the subsection _Light Client Definition_, might be considered in the `Network` category.  So far I think we've assumed that this category is only intended for the Cardano Node's network behaviour.  I don't recall this coming up in prior discussion & wanted to include other `Network` contributors like @jpraynaud if possible.

Reply:
Suggestion was dismissed today (as my misunderstanding) since this doesn't pertain to "network" behaviour but rather methods of [i.e., tools for] integrating other protocol activities with the main protocol.